Docs.md

```diff
@@ -322,6 +322,7 @@ submodule. However, you need to install separately:
 
 Extra steps for Mac users:
 1. [XCode](https://developer.apple.com/xcode/)
+2. Openssl 1.1: `brew install openssl@1.1`
```

Comment:
hmm that shouldn't be necessary but I don't doubt it is now... we use https://github.com/briansmith/ring
Could you append something like `(TODO: shouldn't be necessary)`